src/components/Files/FileUploadDialog.tsx

```
@@ -39,6 +40,19 @@
     }
     onOpenChange(open);
   };
+    
+  const [isPdf, setIsPdf] = useState(false);
```

Biome (1.9.4)

[error] Shouldn't redeclare 'isPdf'. Consider to delete it or rename it.

'isPdf' is defined here:


(lint/suspicious/noRedeclare)